src/main/java/step2/domain/LottoIssuer.java

```diff
@@ -17,7 +17,7 @@
     public static List<Lotto> issue(Money money) {
         List<Lotto> LottoList = new ArrayList<>();
 
-        for (int i = 0; i < money.getNumberOfPurchases(LOTTO_PRICE); i++) {
+        for (int i = 0, j = money.getNumberOfPurchases(LOTTO_PRICE); i < j; i++) {
```

Comment:
로또 구매 금액에 대한 VO를 잘 만들어 주셨기 때문에 이제 LottoIssuer가 굳이 Lotto 가격을 관리하고 있을 필요없고 Money 객체가 로또 구매 금액을 알고 있기에 파라미터로 전달하지 않고 바로 물어봐도 되지 않을까요 :) ?
```suggestion
        for (int i = 0; i < money.getNumberOfPurchases(); i++) {
```